src/examples/websocket_server.conf

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="utf8" standalone="yes"?>
+<websocket_server lockfile="/var/tmp/example.lock">
+  <!--
+  <watchdog glider="/opt/local/bin/bt"
+            tracedir="/var/cores/example"/>
+  -->
+  <eventer>
+    <config>
+      <concurrency>4</concurrency>
+      <default_queue_threads>10</default_queue_threads>
+      <default_ca_chain>/etc/default-ca-chain.crt</default_ca_chain>
+      <ssl_dhparam512_file>/var/run/example/dhparam512.txt</ssl_dhparam512_file>
+      <ssl_dhparam1024_file>/var/run/example/dhparam1024.txt</ssl_dhparam1024_file>
```

Comment:
FYI, after https://github.com/circonus-labs/libmtev/pull/84 we will no longer support 512-bit DH params and will support 2048-bit ones, so if you want, maybe s/512/2048/ here.  Not a big deal though.